### PR TITLE
CU-7ega9r Fixing round buttons, font-size and colors

### PR DESCRIPTION
**Pull request by credondocr**
This pr includes just css changes

### CODE REVIEW

**Comment by credondocr** [comment]
Task linked: [CU-7ega9r General- Header: Switcher de idioma no tiene el estilo correspondiente](https://app.clickup.com/t/7ega9r)

File: components/layout/LanguageSelector.js [hunk 1]
```diff
@@ -4,12 +4,15 @@ import { i18n, withTranslation } from '../../i18n';
 
 const Button = styled.button`
   &:hover {
-    background-color: orange;
+    background-color: #E59E2C;
```
Comment (yeco) [inline components/layout/LanguageSelector.js:7]: Don't we have a themes/colors style sheet?

Comment (jcamacho14) [reply components/layout/LanguageSelector.js:7]: `styles/colors.js`

Comment (jcamacho14) [reply components/layout/LanguageSelector.js:7]: BTW, there's another one in `theme/colors.js`. I don't know why, but we should merge them and keep just one. I would say  the `/styles/` one, as the other style file are located there.

Comment (credondocr) [reply components/layout/LanguageSelector.js:7]: Let me check, anyways we must need to use something like themes

**Pull request merged**